ui/de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/metadata/MetadataPresenterImpl.java

```diff
@@ -155,10 +154,11 @@ public void onFailure(Throwable caught) {
             }
         };
 
-        DiskResourceUserMetadata umd =
-                AutoBeanCodex.decode(autoBeanFactory, DiskResourceUserMetadata.class, "{}").as();
-        umd.setAvus(view.getUserMetadata());
-        drService.setDiskResourceMetaData(resource, umd, view.getAvus(), batchAvuCallback);
+        DiskResourceMetadataList umd =
+                AutoBeanCodex.decode(autoBeanFactory, DiskResourceMetadataList.class, "{}").as();
+        umd.setUserMetadata(view.getUserMetadata());
+        umd.setOtherMetadata(view.getAvus());
+        drService.setDiskResourceMetaData(resource, umd, batchAvuCallback);
```

Comment:
Instead of generating a `DiskResourceMetadataList` with the `AutoBeanCodex` from a string, could this be created with `autoBeanFactory.metadataList()` instead?